contract/src/main/java/com/ritense/valtimo/contract/database/QueryDialectHelper.java

```diff
@@ -23,7 +23,9 @@
 
 public interface QueryDialectHelper {
 
-    Expression<?> getJsonValueExpression(CriteriaBuilder cb, Path column, String path);
+    Expression<String> getJsonValueExpression(CriteriaBuilder cb, Path column, String path);
```

Comment:
Is there a reason why this is 2 different methods? Can't we just pass in the string class for the other one and get rid of this method?

Reply:
Yes, the method is no longer needed. But there is one project that uses this method still. So removing it is a breaking change